upb_generator/protoc-gen-upb.cc

```diff
@@ -1128,6 +1129,8 @@ bool ParseOptions(Plugin* plugin, Options* options) {
       options->bootstrap = true;
     } else if (pair.first == "experimental_strip_nonfunctional_codegen") {
       continue;
+    } else if (pair.first == "dllexport_decl") {
```

Comment:
Sorry, I just noticed: why do we need this in the protoc-gen-upb code generator?

This generator only generates inline methods, so should not require dllexport.

Can we remove the changes in this file, and only change protoc-gen-upbdefs??

Reply:
I just took the changes from #14981, so not sure about the intention behind, but I'll remove it for now.